editor/components/writing-flow/index.js

```diff
@@ -39,15 +39,10 @@ import {
 	isInSameBlock,
 } from '../../utils/dom';
 
-/**
- * Browser dependencies
- */
-
-const { DOMRect } = window;
-
 /**
  * Module Constants
  */
+
```

Comment:
Some remains form previous work?

Reply:
I added it intentionally, only because it's meant as a section marker, not a JSDoc for the variables being assigned. Definitely not related to the pull request, but I thought I could sneak it past. Apparently not without being noticed 😄

Reply:
Oh!

Isn't it common to do this then?

```js
/*
 * Module Constants
 */
```

https://make.wordpress.org/core/handbook/best-practices/inline-documentation-standards/javascript/#multi-line-comments

Reply:
Which we could also use for the "dependencies" comments.

Reply:
Technically, yes (though in the specific example, "Module constants" is prone to laziness and ought to at least be accompanied by variable-specific JSDoc).

I was going to make a big argument about how I disagree with the double-vs-single asterisk, but I honestly don't care too much. There is quite a bit of existing code which would need to be refactored.

Where I might disagree and champion for discussion is some implication from the standards that multi-line comments where each line is prefixed with `//` are not valid for inline code. Aside from being far more verbose, I find it hard to disambiguate between a comment and a JSDoc, where only the latter associates a relationship to the immediate following line (not lines, as in a comment describing a sequence of logic), tying back to good code commenting habits of commenting the "what" (JSDoc) from the "why"/"how" (logic flow comments).

Reply:
Makes sense. I don't care that much either. :)

Reply:
Was just wondering.